app/views/shared/forms/_check_boxes.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-form-group">
+  <fieldset class="govuk-fieldset" aria-describedby="open-banking-consent">
+    <div class="govuk-checkboxes">
+      <div class="govuk-checkboxes__item">
+        <%= form.check_box :open_banking_consent, { class: 'govuk-checkboxes__input' }, 'YES', 'NO' %>
+        <%= form.label :open_banking_consent, t('.label.html'), class: 'govuk-label govuk-checkboxes__label' %>
+      </div>
+    </div>
+ </fieldset>
+</div>
```

Comment:
If this is to be a generic bit of code used where checkboxes are required, the field name and the check box options need to be passed into the partial. At the moment, the "shared" partial can only be used for the field `open_banking_consent`.  So something like:
```erb
<div class="govuk-checkboxes">
  <% field_names_and_choices.each do | name , choices|
    <div class="govuk-checkboxes__item">
      <%= form.check_box name, { class: 'govuk-checkboxes__input' }, *choices %>
      <%= form.label :name, t('.label.html'), class: 'govuk-label govuk-checkboxes__label' %>
    </div>
  <% end %>
</div>
```
Then you can call it with:
```
<%= render partial 'shared/forms/check_boxes', field_names_and_choices: {  open_banking_consent: ['YES', 'NO'] } %>
```
Which can be wrapped in a helper as others are.
Also, the form group can be set with the existing helper method.